### PR TITLE
Restore db context helper

### PR DESCRIPTION
**Pull request by jake-martin**
- re-added dbcontext helper with original methods
- made the kepler methods static (like the original methods)
- commented out existing unit tests and integration tests for Artifact Helpers (there are tasks in Jira to fix them later)
- added an empty rap file to the project and the nuspec file for kepler
- added the name of kepler routes to constants class

There is still some commented code and uncalled methods which will be fixed in future PRs.

### CODE REVIEW

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/TestTemplate.cs [hunk 1]
```diff
@@ -6,9 +6,7 @@
 using System;
 using System.IO;
 using System.Reflection;
-using Relativity.Test.Helpers.ArtifactHelpers;
-using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
-using Relativity.Test.Helpers.ArtifactHelpers.Request;
+//using IServicesMgr = Relativity.Test.Helpers.Interface.IServicesMgr;
```
Comment (calimetikcura) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/TestTemplate.cs]: do we need this commented using statement?

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/DocumentHelperIntegrationTests.cs [hunk 2]
```diff
@@ -13,60 +13,60 @@ namespace Relativity.Test.Helpers.NUnit.Integration.ArtifactHelpers
...
 		public void SetUp()
 		{
-			testHelper = new TestHelper(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
-			SuT = new DocumentHelper(new HttpRequestHelper());
-			_workspaceName = $"IntTest_{Guid.NewGuid()}";
-			_servicesManager = testHelper.GetServicesManager();
-			_workspaceId = WorkspaceHelpers.CreateWorkspace.CreateWorkspaceAsync(_workspaceName,
-				SharedTestHelpers.ConfigurationHelper.TEST_WORKSPACE_TEMPLATE_NAME, _servicesManager,
-				SharedTestHelpers.ConfigurationHelper.ADMIN_USERNAME, SharedTestHelpers.ConfigurationHelper.DEFAULT_PASSWORD).Result;
+			//testHelper = new TestHelper(ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
```
Comment (calimetikcura) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/DocumentHelperIntegrationTests.cs]: are we going to fix these commented tests?

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/TestTemplate.cs (hunk 1 above)
Comment (jake-martin) [reply Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/TestTemplate.cs]: I restored this back to the original class that was in master. I'm going to change this class later.

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/DocumentHelperIntegrationTests.cs (hunk 2 above)
Comment (jake-martin) [reply Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/DocumentHelperIntegrationTests.cs]: Yes, but later.

**Pull request merged**